naive_bayes/crossvalidate.py

```diff
@@ -1,70 +1,79 @@
+"""
+Chapter 4. Naive Bayesian Classification
+The script to investigate cross-validation
+"""
+import io
+
 from spam_trainer import SpamTrainer
 from email_object import EmailObject
-import io
-import re
 
-print "Cross Validation"
+print("Cross Validation")
 
 correct = 0
 false_positives = 0.0
 false_negatives = 0.0
 confidence = 0.0
-    
+
+
 def label_to_training_data(fold_file):
-  training_data = []
-  
-  for line in io.open(fold_file, 'rb'):
-    label_file = line.rstrip().split(' ')
-    training_data.append(label_file)
+    training_data = []
```

Comment:
can you use two space tabs? Since this is going into a book the real estate can be somewhat cramped.